e2e_tests/stepDefinitions/ppcs/ppcs.steps.ts

```diff
@@ -0,0 +1,194 @@
+
+Given('a recommendation exists for use in PPCS', () => {
+    const crn = resolveTestCRN()
+
+    getApiAuth().then((authToken) => {
+        recommendationService.createRecommendation(createRecommendationRequest(crn), authToken).then((recommedation) => {
+            ppcsTestData = {
+                crn: crn,
+                recommendationId: recommedation.id
+            }
+            recommendationService.updateRecommendation(ppcsTestData.recommendationId, initialRecomendationRequest(), authToken)
+            recommendationService.updateRecommendationStatus(recommedation.id, startRecallRecommendationStatusRequest(), authToken)
+        })
+    })
+})
+
+Then('the PPCS recommendation is submitted by a PO', () => {
+    pages.taskListConsiderRecall(ppcsTestData.recommendationId)
+
+    cy.pageHeading().should('equal', 'Consider a recall')
+    cy.clickButton('Continue')
+    cy.pageHeading().should('equal', 'Record the consideration in NDelius')
+    cy.clickButton('Send to NDelius')
+
+    cy.clickLink('Continue to make a recommendation')
+    cy.pageHeading().should('equal', 'Discuss with your manager')
+    cy.clickLink('Continue')
+
+    getApiAuth().then((authToken) => {
+        recommendationService.updateRecommendation(ppcsTestData.recommendationId, postPORecallRecommendationRequest(), authToken, ['previousReleases', 'previousRecalls'])
+    })
+})
+
+Then('the PPCS recommendation has had a recall decision recorded', () => {
+    getApiAuth().then((authToken) => {
+        recommendationService.updateRecommendation(ppcsTestData.recommendationId, recordDecisionRecommendationRequest(), authToken)
+    })
+    
+    signOut()
+    loginAndSearchForCrn(UserType.ACO, ppcsTestData.crn)
+    cy.get(`tr[data-qa="row-${ppcsTestData.crn}"] a[data-qa="name"]`).click()
+    // The service root link is also a link with the text Consider a recall...
+    // This makes the typical clickLink unsuitable
+    // another good reason to make these proper link-as-buttons so they can be better idenfied
+    cy.get('a.govuk-button').contains('Consider a recall').click()
+    
+    cy.pageHeading().should('equal', 'Consider a recall')
+    cy.clickLink('Record the decision')
+    cy.clickButton('Send to NDelius')
+    cy.pageHeading().should('contains', 'Decision to recall')
+})
+
+Then('the PPCS recommendation is counter-signed and downloaded', () => {
+    getApiAuth().then((authToken) => {
+        recommendationService.updateRecommendationStatus(ppcsTestData.recommendationId, spoSignatureRequestStatusRequest(), authToken)
+    })
+    pages.taskListCounterSign(ppcsTestData.recommendationId)
+    cy.pageHeading().should('contain', `CRN: ${ppcsTestData.crn}`)
+    cy.pageHeading().should('contain', 'Part A for')
+
+    const lineManagerLabel = 'Line manager countersignature'
+    cy.clickLink(lineManagerLabel)
+    cy.pageHeading().should('equal', 'Enter your telephone number')
+    textInputRandomText('telephone', faker.phone.number)
```

Comment:
Small point: I don't think this field is mandatory, so you could skip it if you want and simply click on the Continue button, or potentially even directly navigate to the countersignature page directly instead of going through the steps above within this `Then`.

Reply:
I think this raises the point of: so what is the "default expected journey" for PPCS? There are not doubt many permutation that it might be nice run over as we develop this out but for now we should understand what our minimum viable journey should look like.
Maybe one to check with the UX team for advice.
// TODO

Reply:
Ah, but this part isn't the PPCS part yet, it's still the Probation side, that's why I was suggesting the shortcut.

Reply:
Yes, these are optional both in the api and in the flow of pages. Removing for simplicity.